Service/ProcessManager.php

```diff
@@ -201,6 +204,38 @@ public function getNewCaseNumber()
         return date('y').'-'.$caseNumber;
     }
 
+    /**
+     * Completes a Process.
+     *
+     * @param Process $process
+     */
+    public function completeProcess(Process $process)
+    {
+        $completedAt = new \DateTime();
+
+        // If it's the first time the process is completed,
+        // we set the originally completed date.
+        if (null === $process->getLastReopened()) {
```

Comment:
It's not obvious if/that using `getLastReopened` here is correct. Should(n't) it be `getCompletedAt`?